ui/app/components/transit-key-actions.js

```diff
@@ -31,6 +31,9 @@ const STARTING_TRANSIT_PROPS = {
   hash_algorithm: 'sha2-256',
   algorithm: 'sha2-256',
   signature_algorithm: 'pss',
+  padding_scheme: null,
```

Comment:
if you want a global default value, then I recommend setting it here instead of using template logic 
```suggestion
  padding_scheme: 'oaep',
```